{{cookiecutter.github_repository}}/generate_db_schema.py

```diff
@@ -0,0 +1,6 @@
+# Standard Library
```

Comment:
can you put this in `/bin` directory and not clutter the root folder of the project.

Reply:
done